ui/embeds/stats.py

```diff
@@ -15,19 +17,29 @@ async def stats_embed(
     bot: "DiscordBot",
     leetcode_id: str,
     display_name: str,
+    display_url: bool,
     extension: StatsCardExtensions,
 ) -> tuple[discord.Embed, discord.File | None]:
-    file = await stats_card(bot, leetcode_id, extension)
+    # use a randomised filename instead of the user's leetcode_id
```

Comment:
To follow the general convention set out in this project, start with a capital letter, and end with a fullstop.